packages/@react-spectrum/s2/package.json

```diff
@@ -30,11 +30,18 @@
       "require": "./style/dist/main.cjs"
     },
     "./icons/*": {
-      "types": "./icon.d.ts",
+      "types": "./icons/*.d.ts",
       "module": "./icons/*.mjs",
       "import": "./icons/*.mjs",
       "require": "./icons/*.cjs"
-    }
+    },
+    "./illustrations/*": {
+      "types": "./illustrations/*.d.ts",
```

Comment:
The illustrations currently use the same types (copied by below script) as the workflow icons, which means the `size` prop is not exposed. It gets set via context from the IllustratedMessage component, but do we want to support this for standalone use?

Reply:
i think not, at least initially. they come in 3 sizes already, lets find out if that's enough

Reply:
3 sizes you cannot use because the size prop is not exposed. So only via illustrated message...